ecolift/CLAUDE.md

```diff
@@ -12,21 +12,34 @@ The following repositories contain syntax highlighting and similar for editors /
 - git@github.com:actonlang/vim-acton.git
 - git@github.com:actonlang/vscode-acton.git
 
-The following repositories contain Acton code and should be updated when making language changes:
+The following repositories contain Acton code and should be updated when making language changes.
+The Orchestron application stack consists of device drivers (netconf, netclics,
+actmf), YANG library (acton-yang), the Orchestron framework itself (orchestron)
+and the SORESPO application using the entire stack (sorespo).
+netcli is a separate application that uses netclics, netconf and acton-yang
+packages to translate CLI or NETCONF configuration snippets into Acton code
+suitable for writing Orchestron transforms.
 
-- git@github.com:actonlang/acton-snappy.git
-- git@github.com:actonlang/acton-zlib.git
 - git@github.com:orchestron-orchestrator/acton-yang.git
 - git@github.com:orchestron-orchestrator/orchestron.git
 - git@github.com:orchestron-orchestrator/sorespo.git
+- git@github.com:orchestron-orchestrator/netconf.git
+- git@github.com:orchestron-orchestrator/actmf.git
+- git@github.com:orchestron-orchestrator/netcli.git
+- git@github.com:orchestron-orchestrator/netclics.git
+
+The Telemetrify application stack consists of the Telemetrify application
+(telemetrify) and its supporting protocol libraries.
 
 - git@github.com:telemetrify-collector/telemetrify.git
 - git@github.com:actonlang/acton-grpc.git
 - git@github.com:actonlang/acton-http2.git
+- git@github.com:actonlang/acton-snappy.git
+- git@github.com:actonlang/acton-zlib.git
```

Comment:
I placed the telemetrify related apps here more because they aren't as mature yet, so might not work and so it's better to start with the other ones when ecolifting.

zlib is pretty much feature complete and stable. Same for snappy, so I think they can go higher on the list.